### PR TITLE
RGW|BN : change endpoint for http failing tests

### PR DESCRIPTION
**Pull request by AliMasarweh**
## Contribution Guidelines
- To sign and title your commits, please refer to [Submitting Patches to Ceph](https://github.com/ceph/ceph/blob/main/SubmittingPatches.rst).

- If you are submitting a fix for a stable branch (e.g. "quincy"), please refer to [Submitting Patches to Ceph - Backports](https://github.com/ceph/ceph/blob/master/SubmittingPatches-backports.rst) for the proper workflow.

- When filling out the below checklist, you may click boxes directly in the GitHub web UI.  When entering or editing the entire PR message in the GitHub web UI editor, you may also select a checklist item by adding an `x` between the brackets: `[x]`.  Spaces and capitalization matter when checking off items this way.

## Checklist
- Tracker (select at least one)
  - [ ] References tracker ticket
  - [ ] Very recent bug; references commit where it was introduced
  - [ ] New feature (ticket optional)
  - [ ] Doc update (no ticket needed)
  - [x] Code cleanup (no ticket needed)
- Component impact
  - [ ] Affects [Dashboard](https://tracker.ceph.com/projects/dashboard/issues/new), opened tracker ticket
  - [ ] Affects [Orchestrator](https://tracker.ceph.com/projects/orchestrator/issues/new), opened tracker ticket
  - [x] No impact that needs to be tracked
- Documentation (select at least one)
  - [ ] Updates relevant documentation
  - [x] No doc update is appropriate
- Tests (select at least one)
  - [ ] Includes [unit test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/tests-unit-tests/)
  - [ ] Includes [integration test(s)](https://docs.ceph.com/en/latest/dev/developer_guide/testing_integration_tests/)
  - [ ] Includes bug reproducer
  - [x] No tests

<details>
<summary>Show available Jenkins commands</summary>

- `jenkins retest this please`
- `jenkins test classic perf`
- `jenkins test crimson perf`
- `jenkins test signed`
- `jenkins test make check`
- `jenkins test make check arm64`
- `jenkins test submodules`
- `jenkins test dashboard`
- `jenkins test dashboard cephadm`
- `jenkins test api`
- `jenkins test docs`
- `jenkins render docs`
- `jenkins test ceph-volume all`
- `jenkins test ceph-volume tox`
- `jenkins test windows`
- `jenkins test rook e2e`
</details>

### CODE REVIEW

**Comment by AliMasarweh** [comment]
passing teuthology run https://pulpito.ceph.com/alimasa-2024-07-03_11:16:07-rgw:notifications-wip-alimasa-http-to-kafka-distro-default-smithi/

File: src/test/rgw/bucket_notification/test_bn.py [hunk 1]
```diff
@@ -2957,25 +2957,26 @@ def wait_for_queue_to_drain(topic_name, tenant=None, account=None, http_port=Non
     log.info('waited for %ds for queue %s to drain', time_diff, topic_name)
 
 
-@attr('basic_test')
+@attr('kafka_test')
```
Comment (yuvalif) [inline src/test/rgw/bucket_notification/test_bn.py:2960]: can you please also add an http version of this?
we would need to fix http (eventually), so we should have a test to verify our fix

File: src/test/rgw/bucket_notification/test_bn.py [hunk 2]
```diff
@@ -4668,18 +4710,14 @@ def test_persistent_ps_s3_reload():
     http_server.close()
 
 
-@attr('data_path_v2_test')
-def test_persistent_ps_s3_data_path_v2_migration():
+@attr('data_path_v2_kafka_test')
```
Comment (yuvalif) [inline src/test/rgw/bucket_notification/test_bn.py:4713]: same as: https://github.com/ceph/ceph/pull/58408/files#r1664234921

**Comment by AliMasarweh** [comment]
passing run https://pulpito.ceph.com/alimasa-2024-07-04_07:07:09-rgw:notifications-wip-alimasa-http-to-kafka-distro-default-smithi/ (except for http tests)

**Review by yuvalif — APPROVED** [review]

**Comment by yuvalif** [comment]
> passing run https://pulpito.ceph.com/alimasa-2024-07-04_07:07:09-rgw:notifications-wip-alimasa-http-to-kafka-distro-default-smithi/ (except for http tests)

thanks! these are the expected results
please merge when PR checks are passing

**Comment by AliMasarweh** [comment]
jenkins test make check

**Comment by AliMasarweh** [comment]
jenkins test make check

**Pull request merged**